### ISSUE TITLE
Is there a way to configure inline diff background colors?

### ISSUE THREAD

**Issue by steverandy** [divmain/GitSavvy#54]
The default background colors result in a very low contrast because I'm using a light color scheme.
Is there a way to set these color values in the user config?
If not, would you consider to add in the config?

```python
themeGenerator.add_scoped_style("GitSavvy Added Line", "git_savvy.change.addition", background="#37A832")
themeGenerator.add_scoped_style("GitSavvy Removed Line", "git_savvy.change.removal", background="#A83732")
```

Thanks.

**Comment by jtremback** [comment]
+1 It's very hard to read.
![image](https://cloud.githubusercontent.com/assets/1335122/6549901/dfb447f4-c5d8-11e4-84b3-9296f54e90b8.png)

**Comment by divmain** [comment]
Absolutely. I'll have a fix for this in the next day or so.

**Comment by jtremback** [comment]
Wow thanks for the quick response! Would it be possible to simply use the built-in 'Diff' syntax coloring in Sublime? This way the colors will match whatever theme is active.

**Comment by divmain** [comment]
@steverandy @jtremback - I've just added configuration options for inline-diff colors.  Divining the correct color based on the user's theme proved to be time prohibitive, so I opted for this more explicit approach.

Once this is pushed out with the next release, you'll have to delete the corresponding generated `.tmTheme` files from `Packages/User/GitSavvy` and restart Sublime.  GitSavvy re-uses previously generated themes for performance reasons.
